README.md

```diff
@@ -107,13 +70,22 @@ Packages are encapsulated in their own directory, with one subdirectory for each
     │   ├── resource.json
     │   └── package.json
     └── ...
-
 ```
-_Sample package directory layout._
 
-### Content
 
 #### `package.json`
+|Packaging Version|   |
+|-----------------|---|
+|2.0|required|
+|3.0|required|
+
+Every package in Universe must have a `package.json` file which specifies the high level metadata about the package.
+
+Currently, a package can specify two value for `.packagingVersion` either `2.0` or `3.0`, which version is declared
```

Comment:
Added an "s" and some punctuation:

Currently, a package can specify two value**s** for `.packagingVersion`**,** either `2.0` or `3.0`**;** which version ...